cpp/src/rolling/rolling.cu

```diff
@@ -709,6 +709,15 @@ struct rolling_window_launcher {
   {
     if (input.is_empty()) return empty_like(input);
 
+    CUDF_EXPECTS(default_outputs.type().id() == input.type().id(),
+                 "Defaults column type must match input column.");  // Because LEAD/LAG.
+
+    // For LEAD(0)/LAG(0), no computation need be performed.
+    // Return copy of input.
+    if (0 == static_cast<cudf::detail::lead_lag_aggregation*>(agg.get())->row_offset) {
+      return std::make_unique<column>(input, static_cast<cudaStream_t>(0), mr);
```

Comment:
```suggestion
      return std::make_unique<column>(input, stream, mr);
```

Reply:
Thank you, yes. I have a stream available here.

Reply:
Fixed this as well, thank you.